source/api_documentation/signal_api/index.rst

```diff
@@ -470,6 +470,80 @@ Note: As signals are applied to the call, the response time of the API will incr
   curl -k -H "Content-Type: application/json" -X POST -d '{"search": {"transaction_id": "00000000-00000001"},"signals": [{"name": "Quality Call","value": "true"}], "custom_data": [{"name": "channel", "value": "Paid Search"}],"oauth_token": <YOUR OAUTH TOKEN>}'  https://invoca.net/api/@@SIGNAL_API_VERSION/transactions.json
   curl -k -H "Content-Type: application/json" -X POST -d '{"search": {"transaction_id": "00000000-00000001"},"signals": [{"name": "Appointment Made","partner_unique_id": "1","occurred_at_time": "1440607313","value": "false"}], "custom_data": [{"name": "channel", "value": "Paid Search"}],"oauth_token": <YOUR OAUTH TOKEN>}'  https://invoca.net/api/@@SIGNAL_API_VERSION/transactions.json
 
+New Signal Transactions
+-------------------------------
+**Note**: The following only applies if you have already been migrated to the new Signal Transactions system. If you are using the new Signal Transactions, this only applies to calls that occur **after** the date of migration.
+If you're unsure of whether you are using the new Signal Trasactions, please contact Invoca's Customer Success team.
+
+All Signals and Custom Data within a single request will be grouped together based on the value of the field *partner_unique_id*. There will be a single transaction for each group created.
+These new transactions will have the transaction type *Post Call Event*.
+
+Request Parameter changes:
+
+* The existing fields *partner_unique_id*, *occurred_at_time*, and *revenue* can now be specified at the top-level of a request following the same formatting described above. This will be applied to all signals and custom data in the request.
+* These fields can still be specified inline with each signal, but signals with different values for *partner_unique_id* will be not be grouped together.
+* If specified inline for a specific signal, it will take precedence for that signal over the top-level value.
+* The *revenue* field may not be specified at the top-level and inline with signals. If specified inline with signals, it must be the same for all signals with the same partner unique id. This is necessary to prevent ambiguitiy in revenue application.
+* Custom Data fields will be grouped together into a Post Call Event transaction using the top-level *partner_unique_id* (this may be omitted and will default to the empty string). 
+
+Using the same *partner_unique_id* for all signals is **highly recommended** as doing so will greatly improve the performance of the Signals API, Transactions API, and Reports.
```

Comment:
Slight re-word suggestion:

> Using a single *partner_unique_id* at the top level, such that it will be used for all signals and Custom Data, is **highly recommended.**  This will minimize the number of transactions that show up in Transaction Detail reports and the Transactions API.